clients/src/test/java/org/apache/kafka/clients/producer/KafkaProducerTest.java

```diff
@@ -522,15 +511,16 @@ public void testInterceptorPartitionSetOnTooLargeRecord() throws Exception {
         final Cluster cluster = new Cluster(
             "dummy",
             Collections.singletonList(new Node(0, "host1", 1000)),
-            Arrays.asList(new PartitionInfo(topic, 0, null, null, null)),
-            Collections.<String>emptySet(),
-            Collections.<String>emptySet());
+                Collections.singletonList(new PartitionInfo(topic, 0, null, null, null)),
```

Comment:
Unintended?

Reply:
Yeah it was, interestingly IntelliJ automatically formats it like this (don't know why). :)
I'll add the fix soon.